libs/blocks/table/table.css

```diff
@@ -454,6 +397,78 @@ header.global-navigation {
   margin-right: 16px;
 }
 
+/* Start tablet styles */
+@media (max-width: 899px) {
```

Comment:
Is there a specific reason for going desktop-first and not mobile-first? (I get that these were just copied over for this PR, but still wondering about the decision). I also see on L324 that `min-width` is used, I think mixing the two will lead to confusion.